api/exec/kubernetes_deploy.go

```diff
@@ -85,6 +85,19 @@ func (deployer *KubernetesDeployer) Remove(userID portainer.UserID, endpoint *po
 	return deployer.command("delete", userID, endpoint, manifestFiles, namespace)
 }
 
+// Get all Kubernetes resources in the gaven namespace
+func (deployer *KubernetesDeployer) GetAll(endpoint *portainer.Endpoint, namespace string) ([]portainer.KubernetesDeployment, error) {
```

Comment:
no reason for this to be on the kubeDeployer. just do this in the handler

Reply:
Agree, I will move these code to the handler